magento/pwa-studio
# [PWA-1002] Increase Test Coverage in venia-ui/lib/components/CheckoutPage

## Description

As part of our initiative to increase test coverage, stability and confidence in our project we've identified this folder as having a lower than acceptable level of test coverage.

Acceptance Criteria:

Increase test coverage to an acceptable level, ideally 100%

## Implementation Note
There was a component in here `brainTreeDropIn` that I didn't feel comfortable trying to cover. It relied on a lot of waterfall async work to render that wasn't straight forward, so might be better to tap on the original develop for covering that file. Could also be an opportunity to re-visit and see if patterns we've started coalescing on could be added to this component to help simplify.

## Related Issue





* [[PWA-1002](https://jira.corp.magento.com/browse/PWA-1002)] Increase Test Coverage in venia-ui/lib/components/CheckoutPage

## Acceptance



### Verification Stakeholders

### Specification



### Verification Steps


1. Run tests locally or look at Coveralls reports
2. Verify that CheckoutPage, OrderSummary, and PriceAdjustments coverage improved

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
